### PR TITLE
Created regression_system and updated check_loss

### PR DESCRIPTION
**Pull request by morand-g**
:memo: Changelog

## In malpolon/models/standard_prediction_systems.py: Added a generic RegressionSystem
- Updated hardcoded references to classification
- Removed classification-specific default metric
- Updated loss creation so it can be set from the config file (cf. next section)

## In malpolon/models/utils.py:
- Updated argument expected type to allow loss creation from mapping (from cfg file)
- Added condition to trigger loss creation from mapping.

### CODE REVIEW

File: malpolon/models/utils.py [hunk 1]
```diff
@@ -118,7 +118,9 @@ def check_loss(loss: nn.modules.loss._Loss) -> nn.modules.loss._Loss:
     """
     if isinstance(loss, nn.modules.loss._Loss):  # pylint: disable=protected-access  # noqa
         return loss
-    raise ValueError(f"Loss must be of type nn.modules.loss. "
+    elif isinstance(loss, str):
+        return eval(loss)
```
Comment (tlarcher) [inline malpolon/models/utils.py:122]: We are aiming at removing all calls to `eval()` for security purposes. Please take a look at functions `check_optimizer()`, `check_scheduler()` or `check_metrics()`,  on our new branch "no-more-evals" to get an understanding at what a similar `check_loss()` function should look like

File: malpolon/models/standard_prediction_systems.py [hunk 2]
```diff
@@ -449,3 +449,71 @@ def __init__(
...
+    """Regression task class."""
+    def __init__(
+        self,
+        model: Union[torch.nn.Module, Mapping],
+        loss: Union[torch.nn.modules.loss._Loss, str],
+        optimizer: Union[torch.nn.Module, Mapping] = None,
+        lr: float = 1e-2,
+        weight_decay: float = 0,
+        metrics: Optional[dict[str, Callable]] = None,
+        task: str = 'regression_multilabel',
```
Comment (tlarcher) [inline malpolon/models/standard_prediction_systems.py:464]: Argument not used

File: malpolon/models/standard_prediction_systems.py [hunk 3]
```diff
@@ -449,3 +449,71 @@ def __init__(
             }
 
         super().__init__(model, loss, optimizer, metrics=metrics)
+
+
+class RegressionSystem(GenericPredictionSystem):
+    """Regression task class."""
+    def __init__(
```
Comment (tlarcher) [inline malpolon/models/standard_prediction_systems.py:456]: Some arguments (optimizer kwargs) are not used because incompatible with the proposed default optimizer. I suggest replacing them with the default optimizer's; or simply removing them.

Please update the docstring accordingly too
